plugin/converter/testdata/yaml.template.invalid.yml

```diff
@@ -0,0 +1,6 @@
+kind: template
```

Comment:
should this not be a txt file ?

Reply:
It doesn't really matter as i'm testing the converter logic at this stage, the file extension name check would happen in the handler. I only care about testing the bit of logic https://github.com/drone/drone/pull/3128/files#diff-53a3b4e02181d9a28742c3e46d4120505787eb61138ce0fade533705d460d0deR88 and with the change I made in the handler, should never get to the point where the db has an invalid template